modules/management.am

```diff
@@ -326,34 +333,43 @@ function _nolibfuse() {
 }
 
 # REMOVE
+function _detect_appman_apps() {
+	APPMAN_APPSPATH=$(<"$APPMANCONFIG/appman-config")
+	if ! echo "$APPMAN_APPSPATH" | grep -q "^/"; then
+		APPMAN_APPSPATH="$HOME/$(<"$APPMANCONFIG/appman-config")"
+	fi
+}
+
+function _detect_path_to_remove() {
+	[ -d "$APPMAN_APPSPATH"/"$arg" ] && RMPATH="1" || [ -d "$APPSPATH"/"$arg" ] && RMPATH="1" || RMPATH=""
```

Comment:
you might want to turn this into a if elif else statement

Reply:
well, it was an `if`/`fi` statement as I have shown on Discord... but before publishing my code I have tested also this form and worked, so I have done a one-line command to save space

Reply:
It also makes it harder to read, and also the behavior is actually slightly different than that of a if else statement. see this example: 

![image](https://github.com/user-attachments/assets/8681760d-7fde-4d88-9fba-56f7aace9eda)

Reply:
done https://github.com/ivan-hc/AM/pull/1036/commits/c0253f8fc00303d0a4f84503026dd0a470ad41d7